domains/POAS/events/lpa-access-granted/index.md

```diff
@@ -0,0 +1,31 @@
+---
+name: lpa-updated
+version: 0.0.1
+summary: |
+  The LPA's details have changed
+producers:
+  - opg.poas.lpastore
+consumers:
+  - opg.poas.sirius
+  - opg.poas.makeregister
+owners:
+  - vega
+---
+
+## Context
+
+The LPA has been newly registered
```

Comment:
... so the appropropriate actors from MRLPA must be granted access in Use An LPA.